src/xia2/Modules/Indexer/DialsIndexer.py

```diff
@@ -321,6 +322,14 @@ def _index_prepare(self):
             gain = PhilIndex.params.xia2.settings.input.gain
             if gain:
                 spotfinder.set_gain(gain)
+
+            # set a limit for spot finding which is 25% greater than we
+            # actually trust - can use this to measure overloads
```

Comment:
Is the choice of 25% an arbitrary factor in the right ballpark, or is there a specific motivation?

Reply:
The former - though I noted in the few Eiger sets I looked at there were very few pixels > 5% higher than trusted limit. Would you like better justification for this?

I imposed a limit as e.g. relying on 0xffff or similar could pick up incorrectly assessed mask pixels.